Added a white space between text and link in donate page

dikshashukla455 opened the pull request:
Fixes #1186 

 Changes: 

I have added the white space between 'the processing foundation' in English language and it will not affect the other languages.

dikshashukla455 commented:
@limzykenneth  i updated the PR with the better solution that you referred. Plz kindly review it. Thank you.

Ryuyxx commented on src/data/en.yml:
```diff
@@ -261,7 +261,7 @@ download:
   support-1: >-
     We need your help! p5.js is free, open-source software. We want to make our community as open
     and inclusive as possible. You can support this work by making a donation to
-    the
+    the 
```
@dikshashukla455 
Hi, did you check the [previous PR exchanges](https://github.com/processing/p5.js-website/pull/1156#discussion_r818957292) referenced by limzykenneth?

Trailing whitespace is removed by some editors and must be enclosed in single quotation marks.

Example [src/data/en.yml#L82-L83](https://github.com/processing/p5.js-website/blob/main/src/data/en.yml#L82-L83)

dikshashukla455 replied:
Okay, I got it. I will update it quickly. Thank you for your guidance.

dikshashukla455 replied:
@Ryuyxx I have added single quotation marks around them. You can see the below image of my editor:

![Screenshot (1014)](https://user-images.githubusercontent.com/60606998/161297229-81b502fc-1986-4032-8876-a67e887574da.png)

But the browser shows single quotation marks along with the sentence. You can see the below image of my browser:

![Screenshot (1012)](https://user-images.githubusercontent.com/60606998/161297526-1a4d4741-ccf4-4da1-b559-e05efa8a483e.png)

Ryuyxx replied:
You need to remove `>-`

dikshashukla455 replied:
Oh! yeah, I didn't see that. Now, The problem has been resolved. Thank you.

dikshashukla455 replied:
I have updated it according to @limzykenneth  's reference.

dikshashukla455 commented:
> @limzykenneth i updated the PR with the better solution that you referred. Plz kindly review it. Thank you.



> @limzykenneth i updated the PR with the better solution that you referred. Plz kindly review it. Thank you.

@limzykenneth I have updated my PR according with your reference. Plz kindly review it. Thank you

limzykenneth commented:
Looks good. Thanks!

dikshashukla455 commented:
> Looks good. Thanks!

It's a pleasure to contribute
